cpp/src/unary/math_ops.cu

```diff
@@ -466,13 +518,42 @@ struct LogicalOpDispatcher {
   }
 };
 
+struct FixedPointOpDispatcher {
+  template <typename T, typename... Args>
+  std::enable_if_t<not cudf::is_fixed_point<T>(), std::unique_ptr<column>> operator()(
+    Args&&... args)
+  {
+    CUDF_FAIL("FixedPointOpDispatcher only for fixed_point");
+  }
+
+  template <typename T>
+  std::enable_if_t<cudf::is_fixed_point<T>(), std::unique_ptr<column>> operator()(
+    column_view const& input,
+    cudf::unary_op op,
+    cudaStream_t stream,
+    rmm::mr::device_memory_resource* mr)
+  {
+    // clang-format off
+    switch (op) {
+      case cudf::unary_op::CEIL:  return unary_op_with<T, detail::fixed_point_ceil> (input, stream, mr);
```

Comment:
Why do we need to call out the namespace for `fixed_point_ceil` and not `fixed_point_floor` or `fixed_point_abs`?

Reply:
Ah my bad, I fixed this in the midst of some changes that I ended up `git checkout -- .`-ing. Fixed now.